src/index.js

```diff
@@ -1 +1,36 @@
-import './style.css';
\ No newline at end of file
+// imports
+import './style.css';
+import ScoreLibrary from './manage_scores.js';
+import * as DOM from './dom_manipulation.js';
```

Comment:
- [OPTIONAL] If you've considered the above suggestion, this line will have to be refactored as follows:

```suggestion
import createScore from './dom_manipulation.js';
```